### PR TITLE
Refactor games list

### PR DESCRIPTION
**Pull request by AdamSpannbauer**

### CODE REVIEW

File: src/gamesList.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+const GAMES_LIST = [
+  WaterGunRace,
+  FindTheDude,
+  SpotTheDifference,
+  ProtectTheCenter,
+  MemoryGame,
+];
+
+export default GAMES_LIST;
```
Comment (MarkHale94) [inline src/gamesList.js]: If you wanted to, you can do export const GAMES_LIST = [] to do the assignment and export in a single step

Comment (AdamSpannbauer) [reply src/gamesList.js]: Good eye.  I had initially tried that but `eslint` wasn't a fan and said I should prefer `export default` over a naked `export` (tbh I don't fully know the reasons of why, but my short-lived javascript career has been blindly following the Airbnb style guide through `eslint`)



Then `export default` doesn't work inline when declaring a `const`.  So all of that led to the seemingly superfluous declare and export seen here.

If you can drop some knowledge on why preferring `export default` isn't all that `eslint` is making it out to be then I'm definitely down with the more concise `export` syntax.

Comment (AdamSpannbauer) [reply src/gamesList.js]: Apparently, there's some debate on if `export default` is superior or not: https://github.com/airbnb/javascript/issues/1365

Comment (MarkHale94) [reply src/gamesList.js]: Yeah to my knowledge it's just a style convention for them to prefer export default vs the named export.
Here is a concise stack overflow link that goes over some of the differences as well
https://stackoverflow.com/a/33611943

Comment (MarkHale94) [reply src/gamesList.js]: https://developer.mozilla.org/en-US/docs/web/javascript/reference/statements/export
Mozilla documentation on exports

Comment (AdamSpannbauer) [reply src/gamesList.js]: Yeah I think I have a good grasp on what default vs named does for the code, but I was more interested in why the style guide I'm using disallows it.  Here is a concise argument from someone who is anti-`default`: https://gitlab.com/gitlab-org/frontend/rfcs/-/issues/20#advantages-of-switching-patterns

Think I like their arguments...

Comment (MarkHale94) [reply src/gamesList.js]: Most likely it is just from export/import conventions that existed prior. Im very much in the camp of the anti-default guy myself.

Comment (AdamSpannbauer) [reply src/gamesList.js]: i've converted to a named export and opened #29 to make this a project-wide preference for named exports

Comment (MarkHale94) [reply src/gamesList.js]: Rofl what a perfect issue name

Comment (AdamSpannbauer) [reply src/gamesList.js]: ps ima add you as a contributor to this repo so you can do more admin-y stuff if you'd like (and commit to master if you'd like i guess....)

**Review by MarkHale94 — APPROVED** [review]
Looks good!

**Pull request merged**